tests/Umbraco.Tests.AcceptanceTest/misc/Directory.Build.props

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project></Project>
```

Comment:
Not sure I understand this since AcceptanceTest is not a C# project?

Reply:
Looks like this file was removed in #13069, so I've wrongly re-added this while merging 😇

Reply:
And removing it caused the Windows E2E test to fail, as it was inheriting the properties from the repository root `Directory.Build.props`, which (among other things) enabled package validation. I've re-added the empty file, but included a comment mentioning this!